stock_inventory_lockdown/models/stock_inventory.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2013-2016 Numérigraphe SARL
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class StockInventory(models.Model):
+    _inherit = 'stock.inventory'
+
+    @api.model
+    def _get_locations_open_inventories(self, locations_ids=None):
+        """IDs of locations in open exhaustive inventories, with children"""
+        inventory_domain = [('state', '=', 'confirm')]
+        if locations_ids:
+            inventory_domain.append(('location_id', 'child_of', locations_ids))
+        inventories = self.search(inventory_domain)
+        if not inventories:
+            # Early exit if no match found
+            return []
+        location_ids = inventories.mapped('location_id')
+
+        # Extend to the children Locations
+        location_domain = [
+            ('location_id', 'child_of', location_ids.ids),
+            ('usage', 'in', ['internal', 'transit'])]
+        if locations_ids:
+            location_domain.append(('location_id', 'child_of', locations_ids))
```

Comment:
The domain is already `('location_id', 'child_of', locations_ids)` added above in the list!!

Reply:
<code>location_ids</code> != <code>locations_ids</code>

Reply:
@mreficent I am talking about the [line 27](https://github.com/OCA/stock-logistics-warehouse/pull/299/files/4f391ba257942d07fb874f32b16d2483483bc339#diff-3817be29f13c1c2ff56a0a821a025338R27) the child locations are already appended to the domain list.

Reply:
@darshan-serpent

in line 18: you have locations_ids, added in inventory_domain
in line 27: you have location_ids.ids
in line 30: you have locations_ids, added in location_domain

The three things are different.

Reply:
Oh Yes right, now I noticed, thanks! :smile: